AzureCalling/Controllers/StartCallViewController.swift

```diff
@@ -0,0 +1,148 @@
+    private var displayNameTextField: IconTextField!
+    private var nextButton: FluentUI.Button!
+
+    // MARK: ViewController Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        displayNameTextField.becomeFirstResponder()
+    }
+
+    // MARK: UI layout
+    private func setupUI() {
+        view.backgroundColor = ThemeColor.lightSurfacesPrimary
+        self.navigationItem.title = "Start call"
+        setupLabels()
+        setupDisplayNameTextField()
+        setupNextButton()
+        layoutTextFieldContainer()
+        layoutMainContainer()
+    }
+
+    private func setupLabels() {
+        displayNameLabel = FluentUI.Label(style: .footnote)
+        displayNameLabel.text = "Display name"
+        displayNameLabel.textColor = FluentUI.Colors.textSecondary
+
+        captionLabel = FluentUI.Label(style: .caption1)
+        captionLabel.text = "Name shown the others on the call."
+        captionLabel.textColor = FluentUI.Colors.textSecondary
+    }
+
+    private func setupDisplayNameTextField() {
+        displayNameTextField = IconTextField()
+        displayNameTextField.placeholder = "Enter a name"
+        displayNameTextField.image = UIImage(named: "ic_fluent_person_20_regular")
+        displayNameTextField.imageSize = CGSize(width: 16, height: 20)
+
+        displayNameTextField.delegate = self
+        // Dismiss keyboard if tapping outside
+        let tap = UITapGestureRecognizer(target: self.view, action: #selector(UIView.endEditing))
+        view.addGestureRecognizer(tap)
+
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide), name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+
+    private func setupNextButton() {
+        nextButton = FluentUI.Button.createWith(
+            style: .primaryFilled,
+            title: "Next",
+            action: { [weak self] _ in self?.onNextButtonTapped() }
+        )
+    }
+
+    private func layoutTextFieldContainer() {
+        let displayNameLabelContainer = containerWithHorizontalEqualMargin(contentView: displayNameLabel)
+        displayNameLabel.pinToBottom(withMargin: 4)
+        let captionLabelContainer = containerWithHorizontalEqualMargin(contentView: captionLabel)
+
+        displayNameTextField.fixHeightTo(height: 48)
+
+        displayNameStackView = UIStackView(arrangedSubviews: [displayNameLabelContainer, displayNameTextField, captionLabelContainer])
+
+        displayNameStackView.axis = .vertical
+        displayNameStackView.alignment = .fill
+        displayNameStackView.spacing = 4
+        view.addSubview(displayNameStackView)
+
+        displayNameStackView.expandHorizontallyInSuperView(withEqualMargin: 0)
```

Comment:
The parameter of this call is defaulted to zero so you don't need to add it explicitly.